internal/inspectimage/info_display.go

```diff
@@ -187,12 +187,18 @@ func convertToDisplay(proc launch.Process, isDefault bool) ProcessDisplay {
 	case false:
 		shell = "bash"
 	}
+	var argsToUse []string
+	if proc.Args != nil {
+		argsToUse = proc.Args
+	} else if len(proc.Command.Entries) > 1 {
+		argsToUse = proc.Command.Entries[1:]
+	}
```

Comment:
This is still going to drop args on the floor if the process has both non-overridable and overridable args. We could concatenate them, but then the end-user has no way of knowing what will happen if command-line args are provided (which args are going to be overridden, if any?).

If you'd be amenable to it, I think the most correct way to solve this would be something like:

1) Rename `ProcessDisplay.Args` to `ProcessDisplay.AlwaysArgs` or `ProcessDisplay.NonOverridableArgs`
2) Add `ProcessDisplay.OverridableArgs`
3) Populate `ProcessDisplay` as follows:
* If env `CNB_PLATFORM_API` on the image is >= 0.11 AND `proc.BuildpackID` has Buildpack API >= 0.9:
  * `ProcessDisplay.AlwaysArgs` is `proc.Command.Entries[1:]`
  * `ProcessDisplay.OverridableArgs` is `proc.Args`
* If  env `CNB_PLATFORM_API` on the image is < 0.10 OR `proc.BuildpackID` has Buildpack API < 0.9:
  * `ProcessDisplay.AlwaysArgs` is `proc.Args` (`proc.Command` will only ever have one entry)
  * `ProcessDisplay.OverridableArgs` is empty
* If env `CNB_PLATFORM_API` on the image is = 0.10, we kind of don't know how to display the process because we are missing the buildpack ID and hence we don't know the buildpack API. 
  * If `proc.Command.Entries[1:]` exist, then the buildpack must be newer, so we can treat it like the first case ^^
  * Otherwise, `ProcessDisplay.AlwaysArgs` is `proc.Args` and we throw up a warning that they might actually be overridable

We'd also have to make some changes to where we render the display (see where we call `NewInfoDisplay`). This is maybe more work than you bargained for, and if so, that's totally understandable. In the short-term, something like:
```suggestion
	if len(proc.Command.Entries) > 1 {
		argsToUse = proc.Command.Entries[1:]
	}
	argsToUse = append(argsToUse, proc.Args...)
```
will at least capture all the information, while leaving the end-user in the dark about which args may be overridden.

Reply:
@jjbustamante @kotapeter any opposition to merging this PR with the suggested code change, and creating a separate issue to more precisely describe overridable arguments? The current (broken) state could be pretty confusing.